server/src/database/index.ts

```diff
@@ -0,0 +1,49 @@
+import * as dotenv from 'dotenv';
+import { Sequelize } from 'sequelize-typescript';
+import {
+  Achievement,
+  Competition,
+  Group,
+  InitialValues,
+  Membership,
+  Participation,
+  Player,
+  Record,
+  Snapshot
+} from './models';
+
+dotenv.config({ path: process.env.NODE_ENV === 'test' ? '.env.test' : '.env' });
+
+const sequelize = new Sequelize({
```

Comment:
Why isn't this using the configs from the config.ts on the same folder?

Reply:
I was having some issues with TS screaming at me for something inside Sequelize about it. I just never went back to resolve it. I'll go look at it again.

Reply:
Did what I could to use the config more, but the dialect is still an issue so I had to keep the function for checking env.